.github/workflows/ci.yml

```
@@ -0,0 +1,149 @@
+        with:
+          python-version: '3.9'
+          cache: 'pip'
+      
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements-dev.txt
+      
+      - name: Run security checks with bandit
+        run: bandit -r src/ -x tests/
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)